js/localization/messages/sv.json

```diff
@@ -0,0 +1,264 @@
+        "validation-email-formatted": "{0} är ogiltigt",
+        "validation-mask": "Värdet är ogiltigt",
+
+        "dxLookup-searchPlaceholder": "Minsta antal tecken: {0}",
+
+        "dxList-pullingDownText": "Dra neråt för att uppdatera...",
+        "dxList-pulledDownText": "Släpp för att uppdatera...",
+        "dxList-refreshingText": "Uppdaterar...",
+        "dxList-pageLoadingText": "Laddar...",
+        "dxList-nextButtonText": "Mer",
+        "dxList-selectAll": "Välj alla",
+        "dxListEditDecorator-delete": "Radera",
+        "dxListEditDecorator-more": "Mer",
+
+        "dxScrollView-pullingDownText": "Dra neråt för att uppdatera...",
+        "dxScrollView-pulledDownText": "Släpp för att uppdatera...",
+        "dxScrollView-refreshingText": "uppdaterar...",
+        "dxScrollView-reachBottomText": "Laddar...",
+
+        "dxDateBox-simulatedDataPickerTitleTime": "Välj tid",
+        "dxDateBox-simulatedDataPickerTitleDate": "Välj datum",
+        "dxDateBox-simulatedDataPickerTitleDateTime": "Välj datum och tid",
+        "dxDateBox-validation-datetime": "Värdet måste vara ett datum eller en tid",
+
+        "dxFileUploader-selectFile": "Välj fil",
+        "dxFileUploader-dropFile": "eller släpp filen här",
+        "dxFileUploader-bytes": "byte",
+        "dxFileUploader-kb": "kb",
+        "dxFileUploader-Mb": "Mb",
+        "dxFileUploader-Gb": "Gb",
+        "dxFileUploader-upload": "Ladda upp",
+        "dxFileUploader-uploaded": "Uppladdad",
+        "dxFileUploader-readyToUpload": "Klar att ladda upp",
+        "dxFileUploader-uploadFailedMessage": "Uppladdning misslyckades",
+
+        "dxRangeSlider-ariaFrom": "Från",
+        "dxRangeSlider-ariaTill": "Till",
+
+        "dxSwitch-onText": "PÅ",
+        "dxSwitch-offText": "AV",
+
+        "dxForm-optionalMark": "valfri",
+        "dxForm-requiredMessage": "{0} är nödvändigt",
+
+        "dxNumberBox-invalidValueMessage": "Värdet måste vara ett nummer",
+
+        "dxDataGrid-columnChooserTitle": "Kolumnväljare",
+        "dxDataGrid-columnChooserEmptyText": "Dra en kolumn hit för att dölja den",
+        "dxDataGrid-groupContinuesMessage": "Fortsätter på nästa sida",
+        "dxDataGrid-groupContinuedMessage": "Fortsättning från föregående sida",
+        "dxDataGrid-groupHeaderText": "Gruppera enligt denna kolumn",
+        "dxDataGrid-ungroupHeaderText": "Avgruppera",
+        "dxDataGrid-ungroupAllText": "Avgruppera allt",
+        "dxDataGrid-editingEditRow": "Redigera",
+        "dxDataGrid-editingSaveRowChanges": "Spara",
+        "dxDataGrid-editingCancelRowChanges": "Avbryt",
+        "dxDataGrid-editingDeleteRow": "Radera",
+        "dxDataGrid-editingUndeleteRow": "Ångra radering",
+        "dxDataGrid-editingConfirmDeleteMessage": "Är du säker på att du vill radera denna post?",
+        "dxDataGrid-validationCancelChanges": "Avbryt ändringarna",
+        "dxDataGrid-groupPanelEmptyText": "Dra en kolumnrubrik hit för att dölja den",
```

Comment:
@aenges It's the constant about group panel, not about column chooser. Dragging a column there will group DataGrid by this column. How could we change this constant?

Reply:
"dxDataGrid-groupPanelEmptyText": "Dra en kolumnrubrik hit för att gruppera enligt den kolumnen",